### ISSUE TITLE
{Linked:Bug25598620}[Adaptive Cards> Designer]: Screen reader is not announcing the whole information for the 'James Russell' card.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7713]
### Target Platforms

Other

### SDK Version

1.2

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25145.1011)
Browser: Edge Dev (105.0.1321.0 (Official build) dev (64-bit))
Screen reader: Narrator
URL: https://adaptivecards.io/designer/

**Repro steps:**
1. Turn on the narrator through Ctrl+ windows+ enter key.
2. Open the above URL in edge dev browser.
3. Navigate to the designer tab and activate it.
4. Add the provided JSON on Card payload editor and sample payload editor.
5. Activate the preview mode.
6. Navigate to the 'James Russell' and 'James McElroy' through tab key.
7. Observe the issue.

**Actual Result:**
On the 'James Russell' card, screen reader is not announcing the whole information. Narrator is only announcing as **'James Russell Senior Designer button'.**

**Expected Result:**
Screen reader should announce the whole information for the 'James Russell' card. Narrator should announce as **'James Russell Senior Designer button button'.**

**Notes:**
Same issue repro with 'James McElroy' card.

**Observation:**
Issue repro with NVDA.

**User Impact:**
Screen reader user will get impacted as user will not be able to know the whole information and will not be able to track or perform the action properly.

**MAS reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/03.03.02

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

https://user-images.githubusercontent.com/104483005/181201318-1c1a2942-d1bd-4d5b-a373-14064d072229.mp4

<img width="960" alt="MAS3 3 2 Whole information is not announced by narrator for James Russell card" src="https://user-images.githubusercontent.com/104483005/181201844-712e0e23-8192-4d82-8143-5213132018c0.png">

### Card JSON

```JSON
{
  "type": "AdaptiveCard",
  "version": "1.2",
  "speak": "Choose between James McElroy, James Russell",
  "body": [
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": "48px",
          "items": [
            {
              "type": "Image",
              "size": "medium",
              "style": "person",
              "url": "https://graph.microsoft.com/v1.0/users/jamce@microsoft.com/photo/$value",
              "horizontalAlignment": "center",
              "altText": "JM"
            }
          ],
          "verticalContentAlignment": "center"
        },
        {
          "type": "Column",
          "width": "stretch",
          "items": [
            {
              "type": "TextBlock",
              "size": "medium",
              "weight": "bolder",
              "text": "James McElroy",
              "wrap": true
            },
            {
              "type": "TextBlock",
              "isSubtle": true,
              "text": "SENIOR SOFTWARE ENG LEAD",
              "wrap": true,
              "spacing": "none"
            },
            {
              "type": "TextBlock",
              "isSubtle": true,
              "text": "MSAI_UK_1071",
              "wrap": true,
              "spacing": "none"
            }
          ],
          "spacing": "medium"
        }
      ],
      "selectAction": {
        "type": "Action.Submit",
        "data": {
          "ux:position_ref": 1
        },
        "title": "James McElroy SENIOR SOFTWARE ENG LEAD"
      },
      "spacing": "extraLarge"
    },
    {
      "type": "Container",
      "items": [
        {
          "type": "ColumnSet",
          "columns": [
            {
              "type": "Column",
              "width": "48px",
              "items": [
                {
                  "type": "Image",
                  "size": "medium",
                  "style": "person",
                  "url": "https://graph.microsoft.com/v1.0/users/James.Russell@microsoft.com/photo/$value",
                  "horizontalAlignment": "center",
                  "altText": "JR"
                }
              ],
              "verticalContentAlignment": "center"
            },
            {
              "type": "Column",
              "width": "stretch",
              "items": [
                {
                  "type": "TextBlock",
                  "size": "medium",
                  "weight": "bolder",
                  "text": "James Russell",
                  "wrap": true
                },
                {
                  "type": "TextBlock",
                  "isSubtle": true,
                  "text": "SENIOR DESIGNER",
                  "wrap": true,
                  "spacing": "none"
                },
                {
                  "type": "TextBlock",
                  "isSubtle": true,
                  "text": "MSAI Exp PM_UK_1071_Design",
                  "wrap": true,
                  "spacing": "none"
                }
              ],
              "spacing": "medium"
            }
          ],
          "selectAction": {
            "type": "Action.Submit",
            "data": {
              "ux:position_ref": 2
            },
            "title": "James Russell SENIOR DESIGNER"
          },
          "spacing": "extraLarge"
        }
      ],
      "spacing": "large"
    }
  ]
}
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

Could you please provide us any update regarding bug fixing?

**Comment by anna-dingler** [comment]
Hi @vagpt,

This is a card authoring issue. Any additional information that should be announced with the button can be added to `selectAction` `title` property.

Please let me know if you have any questions - thanks!

**Comment by vagpt** [comment]
Hi @anna-dingler,

Thanks for sharing the information.

We have shared your suggestion to 'Cortana' team. We will let you know whether it is working or not when receive the update from them.
